modules/tfm/common/src/tfm_ioctl_ns_api.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <tfm_platform_api.h>
+#include <tfm_ioctl_api.h>
+
+enum tfm_platform_err_t tfm_platform_read(void *destination, uint32_t addr,
```

Comment:
Just a question: does parameter validation occur in TFM core, or we need to have something here?

Reply:
We need to do validation since the output pointer is hidden in the struct being passed.